monad-transformers/src/main/java/com/github/jorgecastillo/kotlinandroid/data/datasource/remote/MarvelNetworkDataSource.kt

```diff
@@ -81,3 +77,7 @@ private fun <D : SuperHeroesContext> fetchHero(ctx: D,
 
 private fun <D : SuperHeroesContext> liftError(
     it: Throwable) = AsyncResult.monadError<D>().raiseError<List<CharacterDto>>(exceptionAsCharacterError(it))
+
+private fun <D : SuperHeroesContext> liftSuccess(
+    it: List<CharacterDto>): AsyncResult<D, List<CharacterDto>> =
```

Comment:
I would not name a function argument `it` because it has semantics of being the argument of a lambda and may confuse beginners.